Makefile

```diff
@@ -526,22 +526,6 @@ define ci_setup_qemu_riscv
 	@$(MAKE) -C "tools/qemu/build" -j2 || (echo "You might need to install some missing packages" || exit 127)
 endef
 
-define ci_setup_qemu_opentitan
-	$(call banner,CI-Setup: Get OpenTitan boot ROM image)
-	# Download OpenTitan image. The latest image URL is available at
-	# https://storage.googleapis.com/artifacts.opentitan.org/latest.txt
-	# We download a fixed version so that new OpenTitan images do not
-	# unexpectedly change OpenTitan's behavior in our CI.
-	@printf "Downloading OpenTitan boot ROM\n"
-	@pwd=$$(pwd) && \
-		temp=$$(mktemp -d) && \
-		cd $$temp && \
-		curl 'https://storage.googleapis.com/artifacts.opentitan.org/opentitan-earlgrey_silver_release_v5-157-ga28c280bb.tar.xz' \
-			--output opentitan-dist.tar.xz; \
-		tar -xf opentitan-dist.tar.xz; \
-		mv opentitan-earlgrey_silver_release_*/sw/device/boot_rom/boot_rom_fpga_nexysvideo.elf $$pwd/tools/qemu-runner/opentitan-boot-rom.elf
-endef
-
```

Comment:
Do we really want to delete this? Or do you just delete this when iterating locally to speed up iterating on QEMU tests?

Reply:
We no longer use the OpenTitan ROM, so there is no reason to download it any more

Reply:
GitHub sent me a notification email saying you made a comment asking me to upload a new ROM artifact. I can no longer see that comment -- is it no longer relevant? If it is relevant, does it matter which commit/CI run I grab the artifact from?

Reply:
Yep, that was my mistaked. We no longer use the ROM, so we don't need it uploaded